### ISSUE TITLE
Update gene lists to be compatible with GENCODE v39

### ISSUE THREAD

**Issue by jharenza** [federicomarini/annoFuseData#6]
Would prefer these to use both old _and_ new symbols as they are just reference lists:

TCGA fusions: https://github.com/d3b-center/annoFuseData/blob/master/inst/extdata/fusionreference.txt
Reference genes: https://github.com/d3b-center/annoFuseData/blob/master/inst/extdata/genelistreference.txt

cc @migbro 

@dmiller15 can you update these two lists using the tool you wrote?

**Comment by dmiller15** [comment]
Just gave these a quick spin and it turns out both fail as they contain genes that have ambiguous new gene names.
For example:
- In Reference genes: `Error updating gene GPRK7. HGNC has multiple options: ['GRK7', 'MKNK2']`
- In TCGA fusions: `Error updating gene MUM1. HGNC has multiple options: ['IRF4', 'PWWP3A']`

I detailed this phenomenon in the original ticket requesting this tool. Our efforts on this ticket will be blocked until a resolution is put forward.

**Comment by jharenza** [comment]
Ok thanks, let me look into it- are these the only two?

**Comment by dmiller15** [comment]
Those were just the first ones that I came across. A full list of all possible conflicts can be found here: https://github.com/PediatricOpenTargets/ticket-tracker/issues/490#issuecomment-1387549587.

I'll pull the list of all the conflicts we see in these files when I have a chance.

**Comment by jharenza** [comment]
>I'll pull the list of all the conflicts we see in these files when I have a chance. 

Thanks that will be helpful - I will likely have to manually go through the literature for them

**Comment by dmiller15** [comment]
From `fusionreference.txt`:
```
Error updating gene C11orf48. HGNC has multiple options: ['C11orf98', 'LBHD1']
Error updating gene MUM1. HGNC has multiple options: ['IRF4', 'PWWP3A']
Error updating gene NET1. HGNC has multiple options: ['SLC6A2', 'SLC6A5']
Error updating gene ODZ3. HGNC has multiple options: ['TENM1', 'TENM3']
Error updating gene QARS. HGNC has multiple options: ['EPRS1', 'QARS1']
Error updating gene STRA13. HGNC has multiple options: ['BHLHE40', 'CENPX']
```

From `genelistrefernce.txt`:
```
Error updating gene GPRK7. HGNC has multiple options: ['GRK7', 'MKNK2']
Error updating gene NET1. HGNC has multiple options: ['SLC6A2', 'SLC6A5']
Error updating gene GGTA1P. HGNC has multiple options: ['GGTA1', 'GGTA2P']
```

**Comment by jharenza** [comment]
OK, I know this is not ideal but these are reference files and all of the multiple option genes exist, so would it be possible to do a replacement such as this:

gene1--C11orf48 --> 
```
gene1--C11orf48
gene1--C11orf98
gene1--LBHD1
```

This way, we would capture all of the fusions and genes, old and new?

**Comment by dmiller15** [comment]
Wouldn't you not want `gene1--C11orf48` in the output since it is either updated to `C11orf98` or `LBHD1`?

**Comment by jharenza** [comment]
No, because not everyone in the world uses gencode 39, so that would limit annoFuse to only be compatible with 39 and not older gene models

**Comment by dmiller15** [comment]
I see. So that's not what the original script does. This proposal is a new feature.

**Comment by jharenza** [comment]
Sorry! Yes, this is different because it is a reference gene /fusion list, opposed to the actual calls which we _did_ want replaced. I just want to be sure annoFuse is not _dependent_ upon GENCODE v39+

**Comment by jharenza** [comment]
I am going to also ask if you can check this file for any updates in gene symbols please: https://github.com/d3b-center/annoFuseData/blob/master/inst/extdata/pfamDataBioMart.RDS (hopefully not, these are kinases only and well-established 🤞 )

**Comment by dmiller15** [comment]
> I am going to also ask if you can check this file for any updates in gene symbols please: https://github.com/d3b-center/annoFuseData/blob/master/inst/extdata/pfamDataBioMart.RDS (hopefully not, these are kinases only and well-established 🤞 )

Done here: https://cavatica.sbgenomics.com/u/d3b-bixu/d3b-dgd-collab-dev/tasks/067c9c9e-5eb4-441d-983b-d89d70e07bed/

**Comment by jharenza** [comment]
closed with #8 #7
